frontend/src/routes/manage/Video/index.tsx

```diff
@@ -110,6 +117,25 @@ const videoColumns: ColumnProps<Event>[] = [
     },
 ];
 
+const SeriesColumn: React.FC<{ title?: string }> = ({ title }) => {
+    const { t } = useTranslation();
+
+    return (
+        <td css={{
+            "&&": { display: "block" },
+            fontSize: 14,
+            ...ellipsisOverflowCss(3),
+        }}>
+            {title
+                // Todo 1: consider making this a link to the series
+                // Todo 2: improve backend sorting so that videos without series are also grouped
```

Comment:
Mh did you still mean to fix that? The first TODO should be trivial to solve (unless I'm mistaken?) and the second one is maybe also fixed by changing [this line](https://github.com/elan-ev/tobira/blob/6640774e45aaba11f74b0312fc48911fce8e3ddf/backend/src/api/model/event.rs#L737) from `"series"` to `"series.title"`?

Reply:
> second one is maybe fixed by changing [this line](https://github.com/elan-ev/tobira/blob/6640774e45aaba11f74b0312fc48911fce8e3ddf/backend/src/api/model/event.rs#L737) from "series" to "series.title"

Apparently not :/